data/tlg2200/tlg00401/__cts__.xml

```diff
@@ -1,7 +1,7 @@
-<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg2200" xml:lang="grc" urn="urn:cts:greekLit:tlg2200.tlg00401">
+<ti:work xmlns:ti="http://chs.harvardx.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg2200" xml:lang="grc" urn="urn:cts:greekLit:tlg2200.tlg00401">
```

Comment:
reads **harvardx**

Reply:
This may be what's breaking the hook test but am unsure — it's something in this file.